hbp_knowledge/hdac6/min2018.bel

```diff
@@ -0,0 +1,203 @@
+the PSD-enriched fraction were also significantly increased by SIRT1
+deletion (t(11)	2.77, p	0.0183, unpaired t test; Fig. 2C,D)."
+p(HGNC:SIRT1) -> p(HGNC:MAPT)
+UNSET MeSHAnatomy
+
+SET Evidence = "SIRT1
+deletion accelerated premature mortality in tauP301S mice (p
+0.0329, log rank; Fig. 3A)."
+p(MGI:Sirt1) -| path(MESH:"Mortality, Premature")
+
+SET Evidence = "No significant synaptophysin
+loss was detected in tauP301S mice or in mice lacking SIRT1
+alone in the frontal cortex and hippocampus (Fig. 3B,C)."
+SET Anatomy = {"frontal cortex","hippocampal field"}
+p(MGI:Mapt,var("p.Pro301Ser")) cnc p(MGI:Syp)
+p(MGI:Sirt1) cnc p(MGI:Syp)
+
+SET Evidence = "However,
+brain-specific SIRT1 deletion in tauP301S mice resulted in a
+significant loss of synaptophysin in the frontal cortex [F(3,19)
+7.5, p 	 0.0079 (tauP301S; SIRT1 F/F vs Nestin-Cre; tauP301S;
+SIRT1 F/F), ANOVA] and hippocampus [F(3,19) 	 11.83, p
+0.0019 (tauP301S; SIRT1 F/F vs Nestin-Cre; tauP301S; SIRT1
+F/F), ANOVA], indicating that SIRT1 deficiency exacerbated
+tau-mediated synaptic deficits (Fig. 3B,C)."
+composite(p(MGI:Mapt,var("p.Pro301Ser")),p(MGI:Sirt1)) -> p(MGI:Syp)
+p(MGI:Mapt,var("p.Pro301Ser")) -> path(MESH:"Neuronal Plasticity")
+p(MGI:Sirt1) -| path(MESH:"Neuronal Plasticity")
+UNSET Anatomy
+
+SET Evidence = "Interestingly, deletion
+of SIRT1 did not affect the number of neurofibrillary tangles
+significantly, as measured with Gallyas silver staining (Fig. 3D). "
+p(MGI:Sirt1) cnc p(MESH:"Neurofibrillary Tangles")
+
+SET Evidence = "We next assessed the effects of SIRT1 deficiency on taumediated
+behavioral deficits. In the EPM test, tauP301S mice deficient
+in SIRT1, compared with tauP301S mice without the
+deficiency, spent significantly more time in the open arm [F(3,50)
+15.05, p 	 0.0001 (tauP301S; SIRT1 F/F vs Nestin-Cre; tauP301S;
+SIRT1 F/F, ANOVA], suggesting that SIRT1 deletion in tauP301S
+mice exacerbates behavioral disinhibition (Fig. 4A)."
+p(MGI:Sirt1) -> path(EFO:"behavioural disinhibition measurement")
+
+SET Evidence = "Regardless of human
+tauP301S expression, SIRT1 deletion
+led to significantly impaired learning for
+locating a hidden platform [p +/- 0.001
+(SIRT1 F/F vs Nestin-Cre; SIRT1 F/F);
+p 	 0.011 (SIRT1 F/F vs Nestin-Cre;
+tauP301S; SIRT1 F/F), multilevel mixed
+effects model; Fig. 4C] and reduced memory
+retention measured by number of platform
+crossings (Fig. 4D)."
+composite(p(HGNC:MAPT,var("p.Pro301Ser")),p(HGNC:SIRT1)) pos bp(GO:"learning or memory")
+
+SET Evidence = "We showed that AAV-SIRT1-induced
+hippocampal SIRT1 overexpression in
+tauP301S mice significantly lowered the
+levels of ac-K174 in the mossy fiber of the
+left hippocampus, compared with the AAVGFP-
+injected controls (t(21) 	 2.5, p
+0.0231, unpaired t test; Fig. 5C,D)."
+SET MeSHAnatomy = "Mossy Fibers, Hippocampal"
+composite(p(MESH:Dependovirus),p(MGI:Sirt1),p(MGI:Mapt,var("p.Pro301Ser"))) -> p(MGI:Sirt1)
```

Comment:
this kind of description for a transgenic mouse model is misleading, please deleted line 188 and provide information about the transgenic system by annotation and comments